entity-framework/core/managing-schemas/migrations/index.md

```diff
@@ -56,7 +56,20 @@ Three files are added to your project under the **Migrations** directory:
 The timestamp in the filename helps keep them ordered chronologically so you can see the progression of changes.
 
 > [!TIP]
-> You are free to move Migrations files and change their namespace. New migrations are created as siblings of the last migration.
+> You are free to move Migrations files and change their namespace manually. New migrations are created as siblings of the last migration.
+> 
+> Alternatively you can use `-Namespace` (Package Manager Console) or `--namespace` (.NET Core CLI) to specify the namespace at generation time.
```

Comment:
This should not be in a tip. Make a different section.

Reply:
Done in PR #2342